### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by lucayepa**
Note that the modified HelpMessage and HelpMessageCli will require translation changes in Transifex for the keys.

### CODE REVIEW

File: INSTALL [hunk 1]
```diff
@@ -2,4 +2,4 @@ Building Bitcoin
 
 See doc/build-*.md for instructions on building bitcoind,
 the intended-for-services, no-graphical-interface, reference
-implementation of Bitcoin.
\ No newline at end of file
+implementation of Bitcoin.
```
Comment (fanquake) [inline INSTALL]: What exactly does this fix?

File: src/init.cpp [hunk 2]
```diff
@@ -1,5 +1,5 @@
 // Copyright (c) 2009-2010 Satoshi Nakamoto
-// Copyright (c) 2009-2014 The Bitcoin Core developers
+// Copyright (c) 2009-2015 The Bitcoin Core developers
 // Distributed under the MIT software license, see the accompanying
```
Comment (fanquake) [inline src/init.cpp]: Please don't change this date.

**Comment by jonasschnelli** [comment]
ACK `init.cpp` and `bitcoin-cli.cpp` changes (except copyright year header)
NACK `INSTALL.md` and `README.md` changes.
It would also be nice if this could get squashed to 1 commit.

File: README.md [hunk 3]
```diff
@@ -60,7 +60,7 @@ submit new unit tests for old code. Unit tests can be compiled and run (assuming
 
 Every pull request is built for both Windows and Linux on a dedicated server,
 and unit and sanity tests are automatically run. The binaries produced may be
-used for manual QA testing — a link to them will appear in a comment on the
+used for manual QA testing -- a link to them will appear in a comment on the
```
Comment (laanwj) [inline README.md]: NACK on this. Using UTF-8 to pretty up the text is fine in 2015.

**Comment by laanwj** [comment]
Also NACK on breaking up the translation messages. This just gives the translators extra work without gain (and they'll spend more time correlating the various messages to keep track of the context). 

Remember that messages won't have the same width in different languages.

If your intent is to format to a certain column width you could use FormatParagraph in utilstrencodings to do so programmatically without changing a single translation message.

File: INSTALL (hunk 1 above)
Comment (laanwj) [inline INSTALL]: Yes, don't make these kind of pointless changes please.

**Comment by laanwj** [comment]
Closing this. Sorry for that, but we're drowning in pull requests and  have to prioritize.

File: INSTALL (hunk 1 above)
Comment (lucayepa) [inline INSTALL]: It is not a pointless change. That file is used by sysadmin to install the software. And when you are a sysadmin and "cat INSTALL.md" does not bring you on the same line, you think you missed something of the original package.

File: src/init.cpp (hunk 2 above)
Comment (lucayepa) [inline src/init.cpp]: I'm not interested in changing this date (obviously), but may I ask why?

File: README.md (hunk 3 above)
Comment (lucayepa) [inline README.md]: This file is used on servers as well. You "cat", "less" or "view" it. A lot of servers are set on ASCII-only in order to be less buggy while editing configuration files. Call it UX for sysadmin, but all the servers I work with display those chars as spurious chars. If you browse projects as gcc, libc and a lot of others, you find that they are in 2015 too, but they care about the "sysadmin UX" more than we do.

**Comment by lucayepa** [comment]
I'll work on FormatParagraph to not break the strings. Thank you for the hint.

**Comment by lucayepa** [comment]
#5734 with FormatParagraph to not break the strings.

**Pull request closed without merging**